spec/requests/custom_actions_spec.rb

```diff
@@ -91,7 +91,7 @@ def expect_result_to_have_custom_actions_hash
       get api_service_url(nil, svc1)
 
       expect_result_to_have_keys(%w(id href actions))
-      expect(response.parsed_body["actions"].collect { |a| a["name"] }).to match_array(%w(edit button1 button2 button3 add_resource remove_resource remove_all_resources add_provider_vms))
+      expect(response.parsed_body["actions"].collect { |a| a["name"] }).to match_array(%w(edit edit edit button1 button2 button3 add_resource remove_resource remove_all_resources add_provider_vms))
```

Comment:
I wonder if these should just be looking at the posts mainly because the action_identifier defines it as such, so maybe something like:

```
  expect(response.parsed_body["actions"].select { |a| a["method"] == "post" }.collect(&:name).to match_array ...
```

thoughts ?